common/canonical_serialization/src/canonical_deserialize.rs

```diff
@@ -0,0 +1,132 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
```

Comment:
I think it would be useful to have a block comment at the top explaining what all this file does.  Doesn't need to be anything huge, but I generally find those useful when reading unfamiliar code